Clean up and reorganize beat snapping articles (#9842)
ppy/osu-wiki: wiki/Client/Beatmap_editor/Beat_snap_divisor/en.md

---
tags:
  - bsd
---

# Beat snap divisor

![](img/beat_snap_divisor.png "Beat snap divisor in the editor")

> The **beat snap divisor** is a setting that determines the editor's behaviour in quantising [beats](/wiki/Music_theory/Beat) based on the [beat snap](/wiki/Client/Beatmap_editor/Beat_snap) principles. It can be found in the upper-right portion of the editor screen.

Author: totally

> The beat snap divisor corresponds to the granularity of the objects on the [timeline](/wiki/Client/Beatmap_editor/Timelines). The second name of the divisor shows how many equal parts individual beats are split into; a denser beat snap divisor setting allows for more notes to be placed in the same measure of time, and vice versa.

Reviewer: ```suggestion
The beat snap divisor corresponds to the granularity of [hit objects](/wiki/Gameplay/Hit_object) on the [timeline](/wiki/Client/Beatmap_editor/Timelines). The divisor is expressed as a fraction of how many parts a beat should be split into; a denser beat snap divisor setting allows for more notes to be placed in the same measure of time, and vice versa.
```

| 1/2 | Red | ![](/wiki/shared/BSD_1_2.jpg "1/2 snap divisor in the osu!/osu!taiko/osu!catch editor") |  |
| 1/3 | Purple | ![](/wiki/shared/BSD_1_3.jpg "1/3 snap divisor in the osu!/osu!taiko/osu!catch editor") | One third of a beat, or a triplet |
| 1/4 | Blue | ![](/wiki/shared/BSD_1_4.jpg "1/4 snap divisor in the osu!/osu!taiko/osu!catch editor") | One quarter of a beat |
| 1/5 | Yellow | ![](img/1_5_snap_divisor.png "1/5 snap divisor in the osu!/osu!taiko/osu!catch editor") | Extremely uncommon |
| 1/6 | Purple | ![](/wiki/shared/BSD_1_6.jpg "1/6 snap divisor in the osu!/osu!taiko/osu!catch editor") | Double triplet (one sixth of a beat) |
| 1/7 | Yellow | ![](img/1_7_snap_divisor.png "1/7 snap divisor in the osu!/osu!taiko/osu!catch editor") | Extremely uncommon |
| 1/8 | Yellow | ![](/wiki/shared/BSD_1_8.jpg "1/8 snap divisor in the osu!/osu!taiko/osu!catch editor") |  |
| 1/9 | Yellow | ![](img/1_9_snap_divisor.png "1/9 snap divisor in the osu!/osu!taiko/osu!catch editor") | Extremely uncommon |
| 1/12 | Grey | ![](img/1_12_snap_divisor.png "1/12 snap divisor in the osu!/osu!taiko/osu!catch editor") | Extremely uncommon |

> | 1/16 | Grey | ![](img/1_16_snap_divisor.png "1/16 snap divisor in the osu!/osu!taiko/osu!catch editor") | Uncommon |

Reviewer: - the files can be moved to the same folder
- "hit object timeline" is probably better than "osu!/osu!taiko/osu!catch editor"
- why are some comments missing?
- rather than/in addition to rarity, maybe this could mention patterns where these divisors are used (1/8 - typical streams, 1/16 - very fast buzz sliders)

Author: > the files can be moved to the same folder

I don't think they are used anywhere else, and frankly I didn't check if stuff from `shared` was

> "hit object timeline" is probably better than "osu!/osu!taiko/osu!catch editor"

..yeah, didn't put much thought in copying and pasting that

> why are some comments missing?

¯\\\_(ツ)\_/¯ there's simply not much to be said about these specific divisors other than making up subjective things. "a half of a beat", and then what? failinig to see any extra value here (on the contrary, another name for 1/3 or 1/6 is worth mentioning due to it being far less known)

> rather than/in addition to rarity, maybe this could mention patterns where these divisors are used (1/8 - typical streams, 1/16 - very fast buzz sliders)

good point, but again, I don't think there's enough information to add to make proper use of the whole column
